Makefile

```diff
@@ -0,0 +1,20 @@
+MAKEFILE_PATH := $(lastword ${MAKEFILE_LIST})
+ROOT_DIR := $(abspath $(dir ${MAKEFILE_PATH}))
+PROTOBUF_DIR := ${ROOT_DIR}/target/protobuf
+
+.PHONY: protobuf
+protobuf: ${PROTOBUF_DIR}/protoc
+
+${PROTOBUF_DIR}/protoc:
+	mkdir -p ${PROTOBUF_DIR}
+	$(eval TMPDIR := $(shell mktemp -d))
+	git clone https://github.com/google/protobuf.git ${TMPDIR}/protobuf
```

Comment:
check out a specific version of protobuf and not the latest

Reply:
Done. Checking out 3.6.1.